sample/app/http-env-echo-no-loadbalancer.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Definitely agree; I will try to mix #50 into this one to make this easier (and doing one test for all seems better than trying to test the same thing separately a few times, for integration tests?)

How are you currently running the integration tests? Was expecting a makefile command